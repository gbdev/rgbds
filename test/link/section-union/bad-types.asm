IF !DEF(SECOND)
	def TYPE equs "HRAM"
ELSE
	def TYPE equs "WRAM0"
ENDC

SECTION UNION "conflicting types", TYPE
	db

	PURGE TYPE
