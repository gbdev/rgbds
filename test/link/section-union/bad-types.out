error: Section "conflicting types" is defined with conflicting types HRAM and WRAM0
---
error: <stdin>(18):
    Section already exists but with type HRAM
FATAL: <stdin>(18):
    Cannot create section "conflicting types" (1 error)
